fabtests/functional/multi_ep.c

```diff
@@ -390,12 +437,24 @@ int main(int argc, char **argv)
 		case 'v':
 			opts.options |= FT_OPT_VERIFY_DATA;
 			break;
+		case LONG_OPT_SHARE_AV:
+			share_av = true;
+			break;
+		case LONG_OPT_SHARE_CQ:
+			share_cq = true;
+			break;
 		case '?':
 		case 'h':
 			ft_usage(argv[0], "Multi endpoint test");
 			FT_PRINT_OPTS_USAGE("-c <int>",
 				"number of endpoints to create and test (def 3)");
 			FT_PRINT_OPTS_USAGE("-v", "Enable data verification");
+			FT_PRINT_OPTS_USAGE("--share-cq",
```

Comment:
There is another test that toggles shared resources and an existing option for sharing the CQ. They all use "shared" instead of "share" - could you change these to "shared" to align with those please?

Reply:
Updated